backend/src/common/enums/bcws/csv-headers.ts

```diff
@@ -0,0 +1,161 @@
+  {
+    field: 'bcws_personnel_willingess_statement',
+    title: 'Willingness Statement Signed',
+  },
+  { field: 'bcws_personnel_par_q', title: 'ParQ Completed' },
+  {
+    field: 'bcws_personnel_workplace_policy',
+    title: 'Workplace Policy Training',
+  },
+  { field: 'bcws_personnel_orientation', title: 'Orientation' },
+  { field: 'bcws_personnel_travel_preference', title: 'Travel Preference' },
+  {
+    field: 'bcws_personnel_first_choice_section',
+    title: 'First Choice Section',
+  },
+  {
+    field: 'bcws_personnel_second_choice_section',
+    title: 'Second Choice Section',
+  },
+  {
+    field: 'bcws_personnel_third_choice_section',
+    title: 'Third Choice Section',
+  },
+  { field: 'personnel_created_at', title: 'Date Created' },
+  { field: 'personnel_updated_at', title: 'Date Updated' },
+  { field: 'personnel_id', title: 'General Personnel ID (Internal Use)' },
+  { field: 'personnel_first_name', title: 'First Name' },
+  { field: 'personnel_last_name', title: 'Last Name' },
+  { field: 'personnel_primary_phone', title: 'Primary Phone' },
+  { field: 'personnel_secondary_phone', title: 'Secondary Phone' },
+  { field: 'personnel_other_phone', title: 'Other Phone' },
+  { field: 'personnel_email', title: 'Email' },
+  { field: 'personnel_supervisor_first_name', title: 'Supervisor First Name' },
+  { field: 'personnel_supervisor_last_name', title: 'Supervisor Last Name' },
+  { field: 'personnel_supervisor_email', title: 'Supervisor Email' },
+  { field: 'personnel_supervisor_phone', title: 'Supervisor Phone' },
+  { field: 'personnel_union_membership', title: 'Union Membership' },
+  { field: 'personnel_intake_form_id', title: 'Intake Form ID' },
+  { field: 'personnel_driver_licenses', title: 'Driving Licenses' },
+  { field: 'personnel_jobTitle', title: 'Job Title' },
+  { field: 'personnel_ministry', title: 'Ministry' },
+  { field: 'personnel_division', title: 'Division' },
+  {
+    field: 'personnel_emergency_contact_first_name',
+    title: 'Emergency Contact First Name',
+  },
+  {
+    field: 'personnel_emergency_contact_last_name',
+    title: 'Emergency Contact Last Name',
+  },
+  {
+    field: 'personnel_emergency_contact_phone_number',
+    title: 'Emergency Contact Phone',
+  },
+  {
+    field: 'personnel_emergency_contact_relationship',
+    title: 'Emergency Contact Relationship',
+  },
+  { field: 'personnel_employee_id', title: 'Employee ID Number' },
+  { field: 'personnel_paylist_id', title: 'Paylist ID Number' },
+  {
+    field: 'personnel_availability_confirmed_until',
+    title: 'Availability Confirmed Until',
+  },
+  {
+    field: 'personnel_availability_confirmed_on',
+    title: 'Availability Confirmed On',
+  },
+  {
+    field: 'personnel_work_location',
+    title: 'Work Location ID (Internal Use)',
+  },
+  {
+    field: 'personnel_home_location',
+    title: 'Home Location ID (Internal Use)',
+  },
+  { field: 'home_loc_id', title: 'Home Location ID (Internal Use)' },
```

Comment:
I would simplify these by just doing name, region, fire centre, fire zone. No need for id, or the other ids above